include/deal.II/fe/mapping_q_generic.h

```diff
@@ -692,16 +692,15 @@ class MappingQGeneric : public Mapping<dim,spacedim>
                                         const Point<dim> &initial_p_unit) const;
 
   /**
-   * For <tt>dim=2,3</tt>. Append the support points of all shape functions
-   * located on bounding lines of the given cell to the vector @p a. Points
-   * located on the vertices of a line are not included.
+   * Append the support points of all shape functions located on bounding
+   * lines of the given cell to the vector @p a. Points located on the
+   * vertices of a line are not included.
    *
-   * Needed by the @p compute_support_points() function. For <tt>dim=1</tt>
-   * this function is empty. The function uses the underlying manifold object
-   * of the line (or, if none is set, of the cell) for the location of the
-   * requested points.
+   * This method uses the underlying manifold object of the line (or, if none
+   * is set, of the cell) for the location of the requested points. This
+   * method is usually called by compute_mapping_support_points() function.
    *
-   * This function is made virtual in order to allow derived classes to choose
+   * This method is made virtual in order to allow derived classes to choose
```

Comment:
Technically (from the perspective of programming languages), this is correct. But C++ nevertheless talks about "member functions", so I would stick with the original naming.